charts/kube-master/templates/api.yaml

```
@@ -335,16 +335,18 @@ spec:
 {{ toYaml .Values.api.resources | indent 12 }}
         - name: wormhole
           image: {{ include "wormhole.image" . | quote }}
-          args:
-            - wormhole
-            - server
-            - --kubeconfig=/etc/kubernetes/config/kubeconfig
-            - --ca=/etc/kubernetes/certs/apiserver-nodes-ca.pem
-            - --cert=/etc/kubernetes/certs/tls-wormhole.pem
-            - --key=/etc/kubernetes/certs/tls-wormhole-key.pem
-            - --service-cidr={{ .Values.serviceCIDR }}
-            - --api-port={{ .Values.advertisePort }}
-            - --v=5
+          command:
+            - sh
+            - -c
```

final nitpick. We should exit on any errors that happen while copying the cert (e.g. can't be found)
```suggestion
            - -ec
```